File: 1672. Richest Customer Wealth/1672. Richest Customer Wealth.c
int maximumWealth(int** accounts, int accountsSize, int* accountsColSize){
    int max = -1;
    for(int i = 0; i < accountsSize; i++){
        int temp = 0;
        for(int j = 0; j < accountsColSize[i]; j++){
            temp += accounts[i][j];
        }
        max = temp > max ? temp : max;
    }
    return max;
}

File: 1672-Richest Customer Wealth/1672-Richest Customer Wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    let max = -1
    for(let i = 0; i < accounts.length; i++){
        let temp = 0;
        for(let j = 0; j < accounts[i].length; j++){
            temp += accounts[i][j]
        }
        max = temp > max ? temp : max
    }
    return max
};
